apps/api/src/app/shared/commands/base.command.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { expect } from 'chai';
```

Comment:
added basic tests for the `BaseCommand`... Unfortunately, I don't have much experience with `mocha/chai/sinon`, but I hope that these are written in the correct way...